ledger/test-common/BUILD.bazel

```diff
@@ -38,6 +38,7 @@ alias(
         ),
     ]
```

Comment:
The alias does not need to be constructed inside a list (minor and pre-existing).